app/src/main/java/com/xlythe/math/Localizer.kt

```
@@ -0,0 +1,145 @@
+                        .contains("matrix") && f.getName().lowercase(Locale.getDefault())
+                        .contains("separator")
+                ) {
+                    mMap.put(",", context.getString(f.getInt(null)))
+                }
+            }
+            Log.d("Localizer", "strings loaded")
+        } catch (e: IllegalArgumentException) {
+            e.printStackTrace()
+        } catch (e: IllegalAccessException) {
+            e.printStackTrace()
+        } catch (e: ClassNotFoundException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)